src/plugins/chart_expressions/expression_heatmap/public/constants.ts

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+import type { RequiredPaletteParamTypes } from '../common';
+export const DEFAULT_PALETTE_NAME = 'temperature';
+export const FIXED_PROGRESSION = 'fixed' as const;
+export const CUSTOM_PALETTE = 'custom';
+export const DEFAULT_CONTINUITY = 'above';
+export const DEFAULT_MIN_STOP = 0;
+export const DEFAULT_MAX_STOP = 100;
+export const DEFAULT_COLOR_STEPS = 5;
+export const DEFAULT_COLOR = '#6092C0'; // Same as EUI ColorStops default for new stops
```

Comment:
nit: maybe it makes sense to import it the way it is imported here? 
https://github.com/elastic/kibana/blob/main/x-pack/plugins/ml/common/util/group_color_utils.ts#L14

Reply:
Well it is not used so I will delete it :D